REUSE.toml

```diff
@@ -1,3 +1,6 @@
+# SPDX-FileCopyrightText: Ansible Project
+# SPDX-License-Identifier: GPL-3.0-or-later
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
```suggestion
```

I don't think this is correct. REUSE.toml is listed as an uncovered file in https://reuse.software/spec-3.3/ and having a copyright header for a license metadata file doesn't make sense.

Reply:
IMO all files should have a header where it is possible, even if the REUSE spec doesn't care.

Reply:
I don't really have a horse in this race, but my take is... I acknowledge the SPDX tags are neither _required_ nor _necessary_ in `REUSE.toml` (as of REUSE 3.3), but I don't see how it can be _wrong_ to include them. The file is no less (or more) copyrightable or subject to license terms than, say, the `pyproject.toml` or `codecov.yml`.

(Which is to say, tho IANAL, my impression is that all three of those _probably_ fall below the threshhold for copyrightability. But, REUSE still demands tags on the other two.) `¯\_(ツ)_/¯`

Reply:
I'm not a lawyer either and I don't care which files are actually copyrightable, but I prefer *all* non-trivial files to have a copyright header. That avoids potential confusion IMO to "why does this file not have a license assigned to it?".

The only exceptions we have are license files themselves, and changelog fragments, which are short-lived and do not exist in releases (because then they'll have been merged into `changelogs/changelog.yaml`, which has a `.license` file next to it).

I also cannot find any mention in the spec that having a copyright header for REUSE.toml is forbidden.